config.json

```diff
@@ -30,6 +30,12 @@
       "values": [],
       "defaultValue": ""
     },
+    "__BlobNamePrefix": {
```

Comment:
Probably yes. I would just make them extra arguments as they aren't common and as such shouldn't really need to be a first class argument.